src/app/components/Picture/index.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import styles from './index.module.scss';
+import { Breakpoint, BreakpointWidths } from 'types/tailwind';
+
+type Sources = {
+  src: string;
+  media: Breakpoint;
+};
+
+type PictureProps = {
+  src: string;
+  srcSet?: Sources[];
+  title?: string;
```

Comment:
Sorry, I made a mistake in the spec, title should be called alt here, an not be used as title attribute.
(Have adjusted the spec, now.) I don't think we will need a title property, the alt should be enough.

Reply:
np, fixed